### ISSUE TITLE
Accessing a resource scoped configuration without providing a resource is not expected

### ISSUE THREAD

**Issue by segevfiner** [Microsoft/vscode-go#2264]
VS Code 1.30.2
vscode-go from Git master

Steps to Reproduce:
1. Start VS Code and run the "Launch extension" debug configuration.

You get the following warning in the debug console:
```
[ms-vscode.Go] Accessing a resource scoped configuration without providing a resource is not expected. To get the effective value for 'go.coverageDecorator', provide the URI of a resource or 'null' for any resource.
```

**Comment by ramya-rao-a** [comment]
This is most likely coming from https://github.com/Microsoft/vscode-go/blob/0.8.0/src/goCover.ts#L53
We can pass the uri of the current active file or fallback to the uri of the first workspace.

PRs are welcome.

**Comment by ramya-rao-a** [comment]
Go ahead @keremgocen :)

**Comment by ramya-rao-a** [comment]
Fixed with #2299
